<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Character_selector</class>
 <widget class="QWidget" name="Character_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="character_list"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="import_button">
         <property name="text">
          <string>Import...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="export_button">
         <property name="text">
          <string>Export...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1">
        <widget class="QSpinBox" name="str_spinbox"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>PP</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Con</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Str</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QSpinBox" name="exp_spinbox"/>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="hp_spinbox"/>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="speed_spinbox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>AC</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QSpinBox" name="cr_spinbox"/>
       </item>
       <item row="16" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Prof</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="wis_spinbox"/>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="pp_spinbox"/>
       </item>
       <item row="13" column="1">
        <widget class="QSpinBox" name="initmod_spinbox"/>
       </item>
       <item row="9" column="1">
        <widget class="QSpinBox" name="cha_spinbox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Speed</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Dex</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Exp</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Int</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Cha</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="int_spinbox"/>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="con_spinbox"/>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="ac_spinbox"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Wis</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Init Mod</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>CR</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="dex_spinbox"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="name_edit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>HP</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QSpinBox" name="proficiency_spinbox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="add_to_character_list">
         <property name="text">
          <string>Add to List</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QListWidget" name="actions_list"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="add_action_button">
         <property name="text">
          <string>Add Action</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>37</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="remove_action_button">
         <property name="text">
          <string>Remove Action</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>import_button</tabstop>
  <tabstop>export_button</tabstop>
  <tabstop>name_edit</tabstop>
  <tabstop>ac_spinbox</tabstop>
  <tabstop>hp_spinbox</tabstop>
  <tabstop>speed_spinbox</tabstop>
  <tabstop>str_spinbox</tabstop>
  <tabstop>dex_spinbox</tabstop>
  <tabstop>con_spinbox</tabstop>
  <tabstop>int_spinbox</tabstop>
  <tabstop>wis_spinbox</tabstop>
  <tabstop>cha_spinbox</tabstop>
  <tabstop>cr_spinbox</tabstop>
  <tabstop>exp_spinbox</tabstop>
  <tabstop>pp_spinbox</tabstop>
  <tabstop>add_to_character_list</tabstop>
  <tabstop>actions_list</tabstop>
  <tabstop>add_action_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
